community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,123 @@
+ - **Be collaborative**: The success and spirit of SourceCred depends on the participation of many people. Effective and friendly collaboration enables everyone to better achieve their goals.  
+ - **Be aware**: We are careful and respectful in our communication, and we take responsibility for our own speech. Be aware of how what you say or do might impact others. Be kind to others. Do not insult or put down other members of the community.
+
+<br>
+
+## Unacceptable Behavior
+
+We, the members of the SourceCred community, are committed to making participation in this community a harassment-free experience.
+
+We will not accept harassment or other exclusionary behaviors, such as:
+ * The use of sexualized language or imagery  
+ * Excessive profanity (please avoid curse words; people differ greatly in their sensitivity to swearing)  
+ * Posting sexually explicit or violent material
+ * Violent or intimidating threats or language directed against another person or group
+ * Inappropriate physical contact and/or unwelcome sexual attention or sexual comments
+ * Sexist, racist, or otherwise discriminatory jokes and language
+ * Trolling or insulting and derogatory comments
+ * Written or verbal comments which have the effect of excluding people on the basis of membership in a specific group, including level
+of experience, gender, gender identity and expression, sexual orientation, disability, neurotype, personal appearance, body size, race,
+ethnicity, age, religion, or nationality
+ * Public or private harassment
+ * Continuing to initiate interaction (such as photography, recording, messaging, or conversation) with someone after being asked to stop
+ * Sustained disruption of talks, events, or communications, such as heckling of a speaker
+ * Publishing (or threatening to post) other people’s personally identifying information (“doxing”), such as physical or electronic
+addresses, without explicit permission
+ * Other unethical or unprofessional conduct
+ * Advocating for, or encouraging, any of the above behaviors
+
+<br>
+
+## Reporting Guidelines
+
+If you believe someone is violating the code of conduct, please report this in a timely manner (see How to Submit a Report). Code of conduct violations reduce the value of the community for everyone.
+
+The SourceCred community takes reports of misconduct very seriously
+and are committed to preserving and maintaining the welcoming nature of our community.
+All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. All reports will be kept confidential, with the exception of cases where the Benevolent Dictator determines the report should be shared with law enforcement. In those cases, the report will be shared with the proper legal authorities.
```

Comment:
> All reports will be kept confidential, with the exception of cases where the Benevolent Dictator determines the report should be shared with law enforcement.

From experience, a moment where such confidential information could easily leak is through the action taken. Such as the previously mentioned "advisory committee from the community" or when providing a reason for a warning / ban. Often by unclear expectations (is it OK to share with the committee how it made the reporter feel?) or accidentally revealing details.

Perhaps they should reference each other. Maybe including a header so that it's more easy to reference.

> ## Reporting Guidelines
> [...]
> ### Confidentiality
> [...]
> ### How to Submit a Report
> [...]

In the Introduction & Scope linking to Confidentiality to make sure people are reminded of this when talking about the actions that can be taken. And here to make sure how to treat a confidential report in such cases.

Reply:
okay I added the sections and did a link from the Introduction and scope, I'll likely need further guidance about what you have in mind for the dual linking.

Reply:
Thanks :smile: I like the new section. I'll add inline comments for easier reference.